net-misc/stargazer/metadata.xml

```diff
@@ -1,54 +1,50 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm sorry but it seems that I've created a merge conflict here. Could you rebase, please?